package.json

```diff
@@ -5,6 +5,7 @@
   "main": "lib/index.js",
   "dependencies": {
     "bluebird": "3.5.0",
+    "deep-equal": "^1.0.1",
```

Comment:
We do not need a dependency for such a simple task.
If you need new generic functionality, first check if a method for that exists in `Util.ts`. If not, add the required method in there.